x/operator/types/utils.go

```diff
@@ -25,3 +25,13 @@ func ChainIDWithLenKey(chainID string) []byte {
 		[]byte(chainID),
 	)
 }
+
+func GetSpecifiedVotingPower(operator string, votingPowerSet []*OperatorVotingPower) *OperatorVotingPower {
+	for idx := range votingPowerSet {
```

Comment:
Why not use pointers to traverse and avoid copying? 

> for _, vp := range votingPowers {
        if vp.OperatorAddr == operator {
            return vp
        }
    }

Reply:
thanks, it will be changed in the next commit.